src/seo/utils/mapLanguagesToLocales.ts

```diff
@@ -0,0 +1,31 @@
+import type { NewsroomLanguageSettings } from '@prezly/sdk';
+
+import type { LangCode, LocaleCode } from '../../intl';
+import { LocaleObject } from '../../intl';
+
+/**
+ * Group languages by their global language code (e.g. "en" for "en-US" and "en-GB").
+ */
+export function mapLanguagesToLocales(languages: NewsroomLanguageSettings[]) {
+    let defaultLangLocale: LocaleObject | undefined;
```

Comment:
There is already `getDefaultLanguage` method, why would you need to duplicate its logic here?
Especially since the method is called `mapLanguagesToLocales`, it's not really suiting for it to also get the default language.

Reply:
Just not to loop the same array twice

Reply:
There would not be enough entries in the `languages` array to make this optimization noticeable, so I would vote for simplicity and readability in this case.

Reply:
Fixed https://github.com/prezly/theme-kit-nextjs/pull/215/commits/ff561de9b1f92e565e8e70d79101def5a19d11d8